---
title: In-silico Construction of pPICZAwbe and pPIC3.5Kwbe
description: In-silico construction of pPICZAwbe and pPIC3.5Kwbe through the elimination of the α-factor signal peptide coding region from pPICZα A and pPIC9K, respectively. 
date: 2022-01-17T07:25:58.471Z
---
| DOCUMENT TYPE | PIC | RESEARCH PERIOD |
| --- | --- | --- |
| THE SECOND SEMESTER REPORT | MUHAMMAD GHILDAN, YEPY RUSTAM | July – December 2021 |

### BACKGROUND

This experiment is aimed to construct pPICZAwbe and pPIC3.5Kwbe to be used for intracellular expression of P. pastoris transporter genes by removing the α-factor signal peptide sequence of pPICZα A and pPIC9K, respectively. The suggested method is based on the overlap-extension PCR (OE-PCR) through the amplification of the initial plasmids using the high-fidelity polymerase and a pair of primers designed to eliminate the α-factor signal peptide coding region while adding a short homologous sequence to both PCR product ends. The resulting linearized plasmids harboring a short homologous sequence at both ends is then directly transformed into E. coli DH5α competent cells to allow for homologous recombination to form a circular plasmid in vivo. Alternatively, the PCR product can be digested with EcoRI and self-religated using T4 DNA Ligase. The resulting plasmids are pPICZAwbe and pPIC3.5Kwbe having the sequence identical to the pPICZ B and pPIC3.5K, respectively. 

### OBJECTIVES

- To provide an experimental design of the construction of pPICZAwbe and pPIC3.5Kwbe by removing the α-factor signal peptide sequence of pPICZα A and pPIC9K, respectively.
- To provide the step-by-step workflow and verification methods.

### MATERIALS AND METHODS

The annotated DNA sequence of Pichia pastoris expression vectors pPICZα A and pPIC9K were obtained from [Addgene vector database] (www.addgene.org). The maps of pPICZα A, and pPIC9K are shown in figure 1 and 9. The more detailed information of the plasmids used in this study is listed in table 1.

Table 1. List of genes and plasmid used or designed in this study.
| Name | Type | Description |
| --- | --- | --- |
| pPICZα A | Expression Vector | P. pastoris expression vector. The vector contains AOX1 promoter, α-factor signal peptide, Myc epitope tag, 6xHis tag, and AOX1 terminator flanking a cloning site. |
| --- | --- | --- |
| pPIC9K |Expression Vector |P. pastoris expression vector capable of multi-gene expression. The vector contains AOX1 promoter, α-factor signal peptide, and AOX1 terminator flanking a cloning site. |
| --- | --- | --- |
| pPICZAwbe | Modified Expression Vector |P. pastoris expression vector. The vector sequence is identical to pPICZ B and similar to pPICZα A but lacking the α-factor signal peptide. |
| --- | --- | --- |
| pPIC3.5Kwbe | Modified Expression Vector | P. pastoris expression vector capable of multi-gene expression. The vector sequence is identical to pPIC3.5K and similar to pPIC9K but lacking the α-factor signal peptide. |

In-silico DNA sequence manipulation and visualization was performed using [Geneious v11.1.5 software](www.geneious.com), which included virtual DNA cloning (restriction and ligation), PCR simulation, sequence editing, sequence alignment, in-silico translation, and sequence annotation. Pairwise and multiple sequence alignment was performed with Global alignment with free end gaps method using Geneious Alignment algorithm. Primers were designed using the in-fusion cloning (Takarabio). Primers’ Tm were calculated using the Geneious software with SantaLucia 1998 formula. Extension of the Primer sequence was performed manually when needed. The primers were purchased from Macrogen Inc. (South Korea).

### RESULTS AND DISCUSSION

Experimental design to construct pPICZAwbe and pPIC3.5Kwbe by removing the α-factor signal peptide coding sequence from pPICZα A and pPIC9K using the  digestion-ligation method is described in the following sections.
The overlap-extension PCR (OE-PCR) method is used to amplify the desired region of plasmid template while removing the unwanted region, i.e., the α-factor signal peptide sequence. Each primer was extended with the homologous sequence to facilitate an in-vivo plasmid circularization in E. coli host cells after transformation. Therefore, the digestion and ligation steps in steps 0A02, 0A03, 0B02 and 0B03 can be be skipped by directly transform the PCR product (i.e., linear plasmid with homologous sequence at both ends) into E. coli host cells.

Elimination of α-factor signal peptide coding region from pPICZα A

The map of the initiating vector pPICZα A is shown in figure 1. The α-factor signal peptide sequence will be eliminated using the overlap-extension PCR (OE-PCR) method. 
