---
title: IN-SILICO CONSTRUCTION OF pPICZAwbe and pPIC3.5Kwbe
description: In-silico construction of pPICZAwbe and pPIC3.5Kwbe through the elimination of the α-factor signal peptide coding region from pPICZα A and pPIC9K, respectively. 
date: 2022-01-17T07:25:58.471Z
---
| DOCUMENT TYPE | THE SECOND SEMESTER REPORT |
|---|---|
| PIC | MUHAMMAD GHILDAN, YEPY RUSTAM |
|---|---|
| RESEARCH PERIOD | July – December 2021 |
|---|---|

### BACKGROUND

This experiment is aimed to construct pPICZAwbe and pPIC3.5Kwbe to be used for intracellular expression of P. pastoris transporter genes by removing the α-factor signal peptide sequence of pPICZα A and pPIC9K, respectively. The suggested method is based on the overlap-extension PCR (OE-PCR) through the amplification of the initial plasmids using the high-fidelity polymerase and a pair of primers designed to eliminate the α-factor signal peptide coding region while adding a short homologous sequence to both PCR product ends. The resulting linearized plasmids harboring a short homologous sequence at both ends is then directly transformed into E. coli DH5α competent cells to allow for homologous recombination to form a circular plasmid in vivo. Alternatively, the PCR product can be digested with EcoRI and self-religated using T4 DNA Ligase. The resulting plasmids are pPICZAwbe and pPIC3.5Kwbe having the sequence identical to the pPICZ B and pPIC3.5K, respectively. 

### OBJECTIVES

- To provide an experimental design of the construction of pPICZAwbe and pPIC3.5Kwbe by removing the α-factor signal peptide sequence of pPICZα A and pPIC9K, respectively.
- To provide the step-by-step workflow and verification methods.

### MATERIALS AND METHODS

The annotated DNA sequence of Pichia pastoris expression vectors pPICZα A and pPIC9K were obtained from [Addgene vector database] (www.addgene.org). The maps of pPICZα A, and pPIC9K are shown in figure 1 and 9. The more detailed information of the plasmids used in this study is listed in table 1.
